catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSource.java

```diff
@@ -763,6 +775,30 @@ private GetFeatureType buildGetFeatureRequest(
           if (areAnyFiltersSet(filter)) {
             wfsQuery.setFilter(filter);
           }
+          if (!this.disableSorting) {
+            if (query.getSortBy() != null) {
+              SortByType sortByType = buildSortBy(filterDelegateEntry.getKey(), query.getSortBy());
+              if (sortByType != null
+                  && sortByType.getSortProperty() != null
+                  && sortByType.getSortProperty().size() > 0) {
+                LOGGER.debug(
+                    "Sorting using sort property [{}] and sort order [{}].",
+                    sortByType.getSortProperty().get(0).getPropertyName(),
+                    sortByType.getSortProperty().get(0).getSortOrder());
+                wfsQuery.setSortBy(sortByType);
+              } else {
+                throw new UnsupportedQueryException(
```

Comment:
Note: I thought it most appropriate to fail the query if the sort property can't be mapped.  GeoServer fails any query with a SortBy property that it doesn't know about.